archlinuxcn/authy-snap/lilac.yaml

```diff
@@ -0,0 +1,12 @@
+maintainers:
+  - github: tonkku107
```

Comment:
Who's tonkku107?

If you send this pull request, it means you're among the maintainers and responsible for packaging issues (listed maintainers will receive error report and be assigned for related github issues). It's not the same as the original PKGBUILD author.

Reply:
He is the guy who owns the AUR Package. This is a copy and paste of the AUR PKGBuild.